api/environments/permissions/migrations/0008_add_manage_segment_overrides_permission.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.2.20 on 2023-11-01 19:54
+
+from django.db import migrations
+from environments.permissions.constants import MANAGE_SEGMENT_OVERRIDES, UPDATE_FEATURE_STATE
+from core.migration_helpers import create_new_environment_permissions
+
+from permissions.models import ENVIRONMENT_PERMISSION_TYPE
+
+def add_manage_segment_overrides_permission(apps, schema_editor):
+    PermissionModel = apps.get_model("permissions", "PermissionModel")
+    UserEnvironmentPermission = apps.get_model(
+        "environment_permissions",
+        "UserEnvironmentPermission",
+    )
+    UserPermissionGroupEnvironmentPermission = apps.get_model(
+        "environment_permissions",
+        "UserPermissionGroupEnvironmentPermission",
+    )
+    manage_segment_overrides_permission, _ = PermissionModel.objects.get_or_create(
+        key=MANAGE_SEGMENT_OVERRIDES,
+        description="Permission to manage segment overrides in the given environment",
+        type=ENVIRONMENT_PERMISSION_TYPE,
+    )
```

Comment:
I think that, since you changed the logic in the previous migration, we don't technically need the `get_or_create` here? We can just use `create`, right?

Reply:
Yeah, but as discussed, it's easier to keep it this way for local developers.